components/atoms/input/MultiRowInput.tsx

```
@@ -0,0 +1,39 @@
+          'group flex max-h-30 h-30 w-[377px] py-3 pl-3 pr-0.5 rounded-lg border border-transparent bg-border-neutral transition-all duration-100 focus-within:bg-bg-base focus-within:border-primary overflow-hidden',
+          className
+        )}
+      >
+        <div className="h-full w-full textarea-custom-scrollbar">
+          <div
+            ref={ref}
+            className={cn(multiRowInputVariants({ className }))}
+            contentEditable="true"
+            role="textbox"
+            data-placeholder={placeholder}
+            {...props}
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<input type="text">
<textarea>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)